xsl/pretext-common.xsl

```diff
@@ -10934,6 +10943,34 @@ http://andrewmccarthy.ie/2014/11/06/swung-dash-in-latex/
     <xsl:value-of select="concat($percent,'%')" />
 </xsl:template>
 
+<!-- ################# -->
+<!-- String Utilities -->
+<!-- ################# -->
+
+<!-- Find a delimiter: find a character that can be wrapped around a string -->
+<!-- Take a string as one input and a list of characters as another input   -->
+<!-- Return the first character from the list that is not in the string     -->
+<xsl:template name="find-unused-character">
+    <xsl:param name="string"/>
+    <xsl:param name="characters" select="concat($apos,'&quot;|/!?=~')"/>
+    <xsl:choose>
+        <xsl:when test="string-length($characters) &lt; 1">
```

Comment:
Could this be `$characters = ''`?

Reply:
Yes! Also, it looks like it could even just be `test="not($characters)"`, because `$characters` is defined as a string based on the `select` in its `xsl:param`, and the empty string is "false" as a boolean. If testing verifies that is the case, do you have a preference?

Reply:
I believe you are entirely correct about the simple `test="not($characters)"`.  I've made a few mistakes with exactly how "empty" is false (node sets, strings, etc), so I have shied away from it mostly.  So I prefer the comparison with the empty string, and it also reads nicely, i think.